### PR TITLE
Improve modal

### PR DESCRIPTION
**Pull request by solomonmcharo28**
Set spacing between buttons and labels, 
Centered the file input prompter, 
Boldened the text of some of the values

### CODE REVIEW

**Review by Carson21 — COMMENTED** [review]
Can you use tailwindcss instead of setting style? For instance, line 104 in CreateReport.jsx can be `className="mr-4"` instead of `style={{ marginRight: 4px }}`

**Review by jonnamorrison — DISMISSED** [review]
would mitigate the bolding of the Upload CSV and query specification labels, would bold create new report label and keep black, submit button should remain consistent with settings one

**Review by Carson21 — CHANGES_REQUESTED** [review]
Use tailwindcss instead of setting style directly

**Comment by p-zach** [comment]
- Modal title should more clearly be a title (a little larger, black text, otherwise it looks like a button with the blue text)
- It still looks a little messy; making it look more like the settings page would probably help. Input forms don't usually have centered text, I think the text should be left-aligned
- None of the other submit buttons in the app have an icon, should probably remove the check mark for uniformity

File: CDC-Data-Reconciliation-Frontend/src/components/CreateReport.jsx [hunk 1]
```diff
@@ -127,7 +130,7 @@ export default function CreateReport({ onDone }) {
             </>
             )}
             <label>
-              <input type='checkbox' checked={isCDCFilter} onChange={handleCDCFilterChange} />
+              <input type='checkbox' checked={isCDCFilter} onChange={handleCDCFilterChange} style={{ marginRight: '4px'}} />
```
Comment (waffy1901) [inline CDC-Data-Reconciliation-Frontend/src/components/CreateReport.jsx:133]: change style={{ marginRight: '4px'}} to className="mr-4"

**Review by Carson21 — APPROVED** [review]
I think it looks way better than before. I think maybe just round the edges a little bit of the background and it will look better.

**Review by jonnamorrison — APPROVED** [review]

**Pull request merged**